app/views/dashboards/_dashboard.html.erb

```diff
@@ -4,7 +4,7 @@
 
 <%= flash_messages %>
 
-<h1><%= "#{t(@dashboard.user_role)}  #{t('dashboard')}" %></h1>
+<h1></h1>
```

Comment:
We probably shouldn't have empty `<h1>`? We should either remove it (which I'm not sure how is it with accessibility) or put something in.